app/javascript/spec/orchestration-template/service-dialog-from-ot.spec.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import fetchMock from 'fetch-mock';
+
+import '../helpers/addFlash';
+import '../helpers/miqFlashLater';
+import '../helpers/miqSparkle';
+import '../helpers/sprintf';
+
+import ServiceDialogFromOt from '../../components/orchestration-template/service-dialog-from-ot';
+
+describe('<ServiceDialogFromOt />', () => {
+  let initialProps;
+
+  beforeEach(() => {
+    initialProps = {
+      otId: 123,
+    };
+  });
+
+  afterEach(() => {
+    fetchMock.reset();
+  });
+
+  it('should submit data correctly', (done) => {
```

Comment:
I wonder about the title here: "should submit..."

Maybe "should fetch", or "should get" ?